site/content/docs/howto/configure-auth-for-webapps.md

```diff
@@ -0,0 +1,347 @@
+---
+title: Using the Pinniped Supervisor to provide authentication for web applications
+description: Allow your Kubernetes cluster users to authenticate into web apps using the same identities.
+cascade:
+  layout: docs
+menu:
+  docs:
+    name: Web Application Authentication
+    weight: 800
+    parent: howtos
+---
+The Pinniped Supervisor is an [OpenID Connect (OIDC)](https://openid.net/connect/) issuer that can be used to bring
+your user identities from an external identity provider into your Kubernetes clusters for all your `kubectl` users.
+It can also be used to bring those same identities to web applications that are intended for use by the same users.
+For example, a Kubernetes dashboard web application for cluster developers could use the Supervisor as its OIDC
+identity provider.
+
+This guide explains how to use the Supervisor to provide authentication services for a web application.
```

Comment:
Maybe this should also make it very clear that this doc is not needed by most users. It is an advanced use case and it is not required for the standard usage of Pinniped, which is to provide auth to users of the `kubectl` CLI (and other clients which use kube client libraries for auth).